src/main/steps/home/get.ts

```diff
@@ -128,8 +129,33 @@ const applicant2RedirectPageSwitch = (req: AppRequest, isFirstQuestionComplete:
   }
 };
 
-const respondentRedirectPageSwitch = (caseState: State, isFirstQuestionComplete: boolean) => {
-  switch (caseState) {
+const respondentRedirectPageSwitch = (req: AppRequest, isFirstQuestionComplete: boolean) => {
+  const nextIncompleteStepUrl = getNextIncompleteStepUrl(req);
+  const hasReviewedTheApplication = !nextIncompleteStepUrl.endsWith(REVIEW_THE_APPLICATION);
```

Comment:
Spoken to Veli about changing this to do a field check rather than using nextIncompleteStepUrl.